api/filter/network/http_connection_manager.proto

```diff
@@ -169,6 +169,10 @@ message HttpConnectionManager {
 
     // Whether to forward the SAN of the client cert. Defaults to false.
     google.protobuf.BoolValue san = 2;
+
+    // [#not-implemented-hide:]
+    // Whether to forward the entire client cert in PEM format. Defaults to false.
+    google.protobuf.BoolValue cert = 3;
```

Comment:
@htuch why did we use the wrapped type for the previous members in this message vs. bool which defaults to false? I don't remember why we ended up doing what we did and naively all of these should be primitive types since they are default false. Did we do this for consistency and because perf doesn't matter? We really should get this written down in STYLE.md. cc @wora